users/src/test/java/io/crate/operation/auth/AuthenticationServiceTest.java

```diff
@@ -65,13 +90,14 @@
 
     @Test
     public void testMissingHbaConf() throws Exception {
-        AuthenticationService authService = new AuthenticationService();
+        AuthenticationService authService = new AuthenticationService(clusterService, settings);
```

Comment:
`authService` could be a field created in `setUpTest`
then also the `settings` field can be removed